source/css/sass/mixins/_typography.scss

```diff
@@ -109,3 +109,16 @@
 @mixin label-tag-typography() {
   @include _label-typography($color-primary-normal);
 }
+
+@mixin teaser-title-text-link-style() {
+
+  .teaser__title_text_link {
```

Comment:
Mixins shouldn't define pattern-specific classes.

There could be a mixin like 'hidden-link' or something. (Like eLife has a [`discreet-link()` mixin](https://github.com/elifesciences/pattern-library/blob/9aad94aef57607c0b2736bbdeee6ed2d74cae3ba/assets/sass/_mixins.scss#L410-L428).)